connector/connect/server/src/main/scala/org/apache/spark/sql/connect/planner/SparkConnectPlanner.scala

```diff
@@ -328,6 +329,16 @@ class SparkConnectPlanner(session: SparkSession) {
       .logicalPlan
   }
 
+  private def transformStatCov(rel: proto.StatCov): LogicalPlan = {
```

Comment:
One of the reasons we haven't implemented that until now is that this performs the execution of the query directly as part of the logical plan.

If you compare this to the other methods they all invoke the DF transformation directly.

One question would be to identify how to rewrite the `cov()` function to be a DF transformation or use it as such.

Reply:
`df.stat.cov` returns a double value, which always trigger a job to compute it.

the users will need to wait for this computation.

Reply:
Yes. So we can collect the result of execution and wraps it with `LocalRelation`.